pkgs/development/tools/coder/default.nix

```
@@ -14,18 +24,58 @@ buildGoModule rec {
     hash = "sha256-26RvDJ890MclDB4rtYQ7CcB3NQRXC7sI2cXd689Eq6E=";
   };
 
-  # integration tests require network access
-  doCheck = false;
+  offlineCache = fetchYarnDeps {
+    yarnLock = src + "/site/yarn.lock";
+    hash = "sha256-xWHLllnJOivjxS56Hp9QhV7oNQhW6TROrzIN1B77eKE=";
+  };
+in
+buildGoModule rec {
+  inherit pname version src;
+
+  subPackages = [ "cmd/coder" ];
```

do we intentionally not want to build `cliui`?

I would prefer `doCheck = false;` and eliding this line (as the effect is the same, the tests are skipped), but if you prefer can we at least:

```suggestion
  subPackages = [ "cmd/..." ];
```

---

Sure I will addopt it :)